parts/k8s/kubernetesbase.t

```diff
@@ -146,54 +146,7 @@
       "location": "[variables('location')]",
       "name": "[variables('nsgName')]",
       "properties": {
-        "securityRules": [
-{{if .HasWindows}}
-          {
-            "name": "allow_rdp",
-            "properties": {
-              "access": "Allow",
-              "description": "Allow RDP traffic to master",
-              "destinationAddressPrefix": "*",
-              "destinationPortRange": "3389-3389",
-              "direction": "Inbound",
-              "priority": 102,
-              "protocol": "Tcp",
-              "sourceAddressPrefix": "*",
-              "sourcePortRange": "*"
-            }
-          },
-{{end}}
-{{if not IsHostedMaster}}
```

Comment:
this whole thing is inside a `{{if IsHostedMaster}}` block https://github.com/Azure/acs-engine/pull/4164/files#diff-66965012f8a909902930dd7fa6804d3eR96 so this is dead code